bottlerocket/agents/src/sonobuoy.rs

```diff
@@ -60,7 +60,7 @@ pub async fn rerun_failed_sonobuoy(
     info!("Rerunning sonobuoy");
```

Comment:
Nit:

For the lack of a better place for this comment,  the message in commit `78880fb52a35d2b498afd1dd0cf5f640cff72cc2` has a typo:


```
Newer versions of sonobuoy no longer uses
```

I think should be:

```
Newer versions of sonobuoy no longer use
```